src/odeproblem.cpp

```diff
@@ -87,7 +87,7 @@ odeproblem::odeproblem(Rcpp::NumericVector param,
   d.amt = 0;
   
   Do_Init_Calc = true;
-  ss_tol = 1e-12;
+  ss_fixed = false;
   ss_n = 1000;
   
```

Comment:
Why is `ss_n` in this constructor different from that in the header? Here we have 1000, in the header it's 500. Is there anywhere `odeproblem` will be constructed using the default constructor?

EDIT: if this is going to be always fixed with a default value, isn't previously defined `MRGSOLVE_MAX_SS_ITER` for this purpose?

Reply:
- Removed the default constructor (wasn't used anywhere)
- Set initialize ss_n to 500 in the (one) constructor
- removed `MRGSOLVE_MAX_SS_ITER` (legacy)